src/Illuminate/Foundation/helpers.php

```diff
@@ -577,16 +577,21 @@ function mix($path, $manifestDirectory = '')
             $manifestDirectory = "/{$manifestDirectory}";
         }
 
+        $manifestKey = $manifestDirectory ? $manifestDirectory : '/';
+
         if (file_exists(public_path($manifestDirectory.'/hot'))) {
             return new HtmlString("//localhost:8080{$path}");
         }
 
-        if (! $manifest) {
+        if (in_array($manifestKey, $manifests)) {
```

Comment:
Shouldn't this be `array_key_exists($manifestKey, $manifests)`?

@mathieutu @GeorgeHanson @taylorotwell